### ISSUE TITLE
Request: Post this on SideQuest

### ISSUE THREAD

**Issue by sigboe** [y-fujii/ovrvnc#6]
Hey, I just got my Quest 2 yesterday, and I realize this app is not on SideQuest, nor is any other VNC app I could find (just tried searching for VNC)
SideQuest is a storefront/website/desktop app, that allows installing and updating programs/addons/patches without going through Oculus Store. Exmaple Virtual Desktop (no linux support) is sold on the official store, but the feature that enabled you to play PC VR games over Wifi was not allowed in the store, and so you buy the app on the Oculus platform, and install the extra feature using SideQuest. Note that this official patch for Virtual Desktop anlone has over 215 thousand downloads via SideQuest.
Games like open source ports of Half-Life 1, Doom1/2 etc, Quake1, Quake 2 and more are also available on SideQuest, just requiring you to add the game asset files your self.

Would be cool to have OVRVNC available on SideQuest, so that maybe more people can try it or keep it updated without downloading files and running adb commands manually.

**Comment by WilsonBradley** [comment]
Agree.. been asking for years... I have to sideload Android VNC clients
